internal/web_server/web/views/components/quiz_components/play_quiz_components.templ/play_quiz_components.templ

```diff
@@ -0,0 +1,128 @@
+	>
+		<form
+			id="answer-form"
+			hx-post={ fmt.Sprintf("/api/v1/quiz/user-answer?question-id=%s", question.ID.String()) }
+			hx-swap="outerHTML"
+			hx-target="#answer-buttons"
+			hx-indicator=".htmx-indicator"
+			class="grid grid-cols-2 gap-8 w-full isolate mt-2"
+		>
+			for _, alternative := range question.Alternatives {
+				<button
+					type="submit"
+					name="answer"
+					class="text-xl p-2 h-28 rounded gradient-shadow bg-white text-wrap w-full"
+					value={ alternative.ID.String() }
+				>
+					{ alternative.Text }
+				</button>
+			}
+		</form>
+		@components.LoadingIndicator()
+		@stopTimerBeforeRequest()
+	</div>
+}
+
+script stopTimerBeforeRequest() {
+	const answerForm = document.getElementById('answer-form');
+	answerForm.addEventListener('htmx:beforeRequest', function(evt) {
+		const timerELem = document.getElementById('timer');
+		timerELem.dispatchEvent(new CustomEvent('quiz:stop-timer'));
+	});
+}
+
+templ FeedbackButtons(answered *user_quiz.UserAnsweredQuestion) {
+	<div class="flex flex-col gap-8 items-center w-full">
+		<div class="grid grid-cols-2 gap-8 w-full isolate mt-2">
+			for _, alt := range answered.Question.Alternatives {
+				@feedbackButton(alt, alt.ID == answered.ChosenAnswerID)
+			}
+		</div>
+		if answered.NextQuestionID != uuid.Nil {
+			<button
+				class="text-xl py-2 px-8 w-min rounded-sm gradient-shadow bg-white text-wrap flex flex-row items-center gap-2"
+				hx-get={ fmt.Sprintf("/api/v1/quiz/next-question/?quiz-id=%s", answered.Question.QuizID.String()) }
+				hx-target="#quiz-play-content"
+				hx-swap="outerHTML"
+				hx-indicator=".htmx-indicator"
+			>
+				NESTE
+				@components.LoadingIndicator()
+			</button>
+		} else {
+			<a
+				class="text-xl py-2 px-8 w-min rounded-sm gradient-shadow bg-white text-wrap"
+				href={ templ.SafeURL(fmt.Sprintf("/quiz/summary?quiz-id=%s", answered.Question.QuizID.String())) }
+			>
+				FERDIG
+			</a>
+		}
+	</div>
+}
+
+templ feedbackButton(alternative questions.Alternative, wasPicked bool) {
+	<button
+		disabled
+		if wasPicked {
+			class="text-xl p-2 h-28 rounded gradient-shadow bg-white text-wrap w-full border-4 border-blue-500"
+		} else {
+			class="text-xl p-2 h-28 rounded gradient-shadow bg-white text-wrap w-full"
+		}
+	>
+		{ alternative.Text }
+		<div class="absolute -top-4 -left-4 ">
+			if alternative.IsCorrect {
+				@feedbackCircle(true, wasPicked)
+			} else {
+				@feedbackCircle(false, wasPicked)
+			}
+		</div>
+		<div class="absolute -bottom-4 -right-4">
+			@feedbackPercentage(alternative.PercentChosen, wasPicked)
+		</div>
+	</button>
+}
+
+templ feedbackCircle(correct bool, wasPicked bool) {
+	<div
+		if wasPicked {
+			class="flex items-center justify-center w-12 h-12 rounded-full bg-white gradient-shadow border-4 border-blue-500"
+		} else {
+			class="flex items-center justify-center w-12 h-12 rounded-full bg-white gradient-shadow"
+		}
+	>
+		if correct {
+			@icons.Checkmark(60, "green", 1, 1)
```

Comment:
These old icons use Tailwind classes, but perhaps the icons should be refactored to use pixels instead.